bucket/saml2aws.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/Versent/saml2aws",
+    "description": "CLI tool which enables you to login and retrieve Amazon Web Services / AWS temporary credentials using a SAML IDP.",
+    "license": "MIT",
+    "version": "2.16.0",
+    "url": "https://github.com/Versent/saml2aws/releases/download/v2.16.0/saml2aws_2.16.0_windows_amd64.tar.gz",
+    "hash": "sha512:f5be1b59cc810c36949aaca3d5eed9a83e4b73d89abef8d5d71580fbb7ee45f4eb662af4c87845c79ec11088c2fb988e810c9cb38c8062618eb81aa972d5f4cc",
+    "bin": "saml2aws.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/Versent/saml2aws/releases/download/v$version/saml2aws_$version_windows_amd64.tar.gz",
+        "hash": {
+            "url": "https://github.com/Versent/saml2aws/releases/download/v$version/saml2aws_$version_windows_amd64.sha512"
```

Comment:
```suggestion
            "url": "$baseurl/saml2aws_$version_windows_amd64.sha512"

```

Reply:
@linsui I've pushed the suggested changes.